pkgs/development/tools/continuous-integration/jenkins-job-builder/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, pythonPackages, buildPythonPackage, git }:
+
+buildPythonPackage rec {
+  name = "jenkins-job-builder-${version}";
+  version = "1.2.0";
+  namePrefix = "";
+
+  src = fetchurl {
+    url = "https://pypi.python.org/packages/source/j/jenkins-job-builder/jenkins-job-builder-${version}.tar.gz";
```

Comment:
I'm not sure I understand what you're asking for here.  Do you mean this?

```nix
{
  name = "1.2.0";
  namePrefix = "jenkins-job-builder";
  src = fetchurl {
    url = "https://.../${namePrefix}-${name}";
    ...
  };
}
```

If so, that seems kinda weird - the namePrefix field is the bit where buildPythonPackage normally adds something like "python27-", and setting it to an empty string simply disables that.  The name/version pattern is what I'm used to seeing in nixpkgs, so I assume I must be misunderstanding you slightly.